xml/operations-maintenance-controller-full_recovery_test.xml

```diff
@@ -543,7 +543,106 @@ ansible-playbook -i hosts/verb_hosts site.yml --limit doc-cp1-c1-m1-mgmt,doc-cp1
        </listitem>
      </orderedlist>
    </section>
-   <section>
+     <section>
+       <title>Cassandra database restore</title>
+       <para>Cassandra database restore</para>
+       <para>Create a script cassandra-restore-extserver.sh on all
+       controller nodes</para>
```

Comment:
all controller nodes where Cassandra runs, which you can determine by running `ansible FND-CDB --list-hosts` on the deployer, from the scratch area.